internal/learn/about-iota/tangle.md

```diff
@@ -1,42 +1,49 @@
 ---
 id: tangle
 title: The Tangle
-description: The Tangle is the data structure behind IOTA and it is different from a blockchain.
+description: The Tangle is IOTA's distributed ledger that contains all the information necessary to track token ownership.
 image: /img/iota-wiki.png
 keywords:
-  - explanation
+- explanation
 ---
 
 # The Tangle
 
-The Tangle is a structure that holds all necessary information to keep track of token ownership. It is a directed acyclic graph (DAG) of messages, where each newer message is attached to two to eight older ones. The protocol can process these various attachments in parallel.
+The Tangle is IOTA's distributed ledger. It's a data structure replicated across a network of computers (also called 'nodes') that contains all the information necessary to track token ownership. It forms a directed acyclic graph of blocks (a block-DAG), where each newer block is attached to multiple older ones.
```

Comment:
So about blocks... Not sure if we should already use that in the IOTA part of the wiki as this is something coming with Shimmer. So it isn't "live" yet on IOTA 🤔

Reply:
Good point. I think calling it further "Message" doesn't help people to understand our tech. And the wiki should be all about understanding, so we shouldn´t wait for Stardust to be finally deployed to the IOTA mainnet.
Also we are talking here about a general Tangle documentation, it should be used for IOTA and Shimmer.

Specific documentation about the "Message" object in IOTA should stay unchanged IMO.

Reply:
Seeing that we'll be separating the IOTA and Stardust wikis, it shouldn't be too much work since most of the stardust docs already use blocks instead of messages. @Dr-Electron, would it be possible to have a `stardust` branch in the IOTA wiki for now so we can isolate this and keep `messages` in the `main` branch and `blocks` in the `stardust` branch?

Reply:
😅 We are probably going down the `separate folder` way. So basically like docusaurus versioning works.
But Sam thinks we could just use blocks in the high level descriptions of IOTA too 🤔

Reply:
So we agree. And can merge. Makes sense 😆